packages/fluentui/react/src/themes/teams/components/Card/cardPreviewStyles.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { TeamsCardVariables } from './cardVariables';
+import { CardPreviewStylesProps } from '../../../../components/Card/CardPreview';
+
+const cardPreviewStyles: ComponentSlotStylesPrepared<CardPreviewStylesProps, TeamsCardVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    return {
+      ...(!p.noMarginAfter && {
+        marginBottom: v.cardChildMarginBottom,
+        ...(p.horizontal && {
+          marginBottom: undefined,
+          marginRight: v.cardChildMarginBottom
+        })
+      })
```

Comment:
```suggestion
      margin: v.previewMargin,
      ...(p.noMarginAfter && {
        margin: v.cardChildMargin,
      }),
      
      ...(p.horizontal && {
          margin: v.horizontalPreviewMargin,
        })
      })
```

Let's avoid negative conditions